src/infinity-menu.js

```diff
@@ -266,7 +278,7 @@ export default class InfinityMenu extends React.Component {
 	 *  @description React render method for creating infinity menu
 	 */
 	render() {
-		const tree = dcopy(this.props.tree);
+		const tree = this.props.tree;
```

Comment:
can you just quickly explain what parts of the changes allowed us to remove deep copy? @tiansijie